src/components/signup-view/signup-view.jsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
+
+export const SignupView = () => {
+    const [username, setUsername] = useState("");
+    const [password, setPassword] = useState("");
+    const [email, setEmail] = useState("");
+    const [birthday, setBirthday] = useState("");
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+
+        const data = {
+            Username: username,
+            Password: password,
+            Email: email,
+            Birthday: birthday
+        };
+
+        fetch("https://myflix-20778.herokuapp.com/login", {
```

Comment:
it should be `/users` instead of `/login`

Reply:
updated

Reply:
It still shows `/login` for me. Can you please check and update it to `/users`